projects/feedtheworld/index.js

```diff
@@ -0,0 +1,16 @@
+const { sumTokens2 } = require('../helper/solana')
+
+module.exports = {
+  solana: {
+    tvl: async (api) => {
```

Comment:
please use `tvl: () => ({})` and export the function here as `staking` as it is your own protocol token